app/main.py

```diff
@@ -1 +1,12 @@
 # write your code here
+def copy_file(cmd: str) -> None:
+    cmd_list = cmd.split()
+    existing_file = cmd_list[1]
+    new_file = cmd_list[2]
```

Comment:
Using unpacking for variable assignment is a concise and more readable approach that, prior to implementation, requires verifying that the length of cmd_list is 3.